docs/get-details/dapps/smart-contracts/guidelines.md

```diff
@@ -0,0 +1,374 @@
+title: Modern guidelines for smart contracts and smart signatures on Algorand
+
+## Disclaimer
+
+The content provided here is not an exhaustive list of security considerations. Before deploying any application in production, it is strongly advised to get the contract audited by one more third-party security companies.
+
+## DO NOT use smart signatures (except for niche use cases)
+
+Algorand supports two types of programs on the blockchain: **smart signatures** and **smart contracts**. Smart signatures themselves can be used both as **delegated (logic) signatures** or **contract account**.
+
+Nowadays, 99% of the dApps should **only** use **smart contracts** and should **not** use any **smart signatures**. 
+
+See the [section "More about smart signatures vs smart contracts"](#more-about-smart-signatures-vs-smart-contracts) for additional details.
+
+!!! info
```

Comment:
drop on the Internet, with on the developer portal or anywhere else online using smart signatures